### ISSUE TITLE
Duplicate code detected in imports

### ISSUE THREAD

**Issue by danijar** [PyCQA/pylint#803]
I would argue that similar import statements in multiple files are fine. On the other hand, I don't want to disable `duplicate-code` in general since it is a useful check.

```
************* Module test.test_default
R:  1, 0: Similar lines in 2 files
==test.test_default:1
==test.test_elements:1
from datetime import date
import pytest
from definitions import Parser
from definitions.error import DefinitionError
from test.fixtures import *

 (duplicate-code)
Command failed with exit code 8
```

**Comment by PCManticore** [comment]
You could try to run it with --ignore_imports=y and it shouldn't complain about this.

**Comment by danijar** [comment]
Wouldn't that disable all kinds of import linting?

**Comment by PCManticore** [comment]
Nope, only similarities with regard imports.

**Comment by PCManticore** [comment]
Let me know if it works for you so I could close the issue.

**Comment by danijar** [comment]
Thanks.
